src/main/flight/interpolated_setpoint.c

```diff
@@ -58,9 +66,36 @@ FAST_CODE_NOINLINE float interpolatedSpApply(int axis, bool newRcFrame, ffInterp
         const float rxInterval = currentRxRefreshRate * 1e-6f;
         const float rxRate = 1.0f / rxInterval;
 
-        const float setpointSpeed = (rawSetpoint - prevRawSetpoint[axis]) * rxRate;
-        const float setpointAcceleration = (setpointSpeed - prevSetpointSpeed[axis]) * pidGetDT();
+        float setpointSpeed = (rawSetpoint - prevRawSetpoint[axis]) * rxRate;
+        float setpointAcceleration = setpointSpeed - prevSetpointSpeed[axis];
+        const float holdSteps = 2.0f;
+
+        // Glitch reduction code for identical packets
+        if (setpointSpeed == 0 && fabsf(rawSetpoint) < 0.98f * ffMaxRate[axis]) {
+            // identical packets, not at full deflection
+            if (holdCount[axis] < holdSteps && fabsf(rawSetpoint) > 2.0f && !bigStep[axis]){
```

Comment:
In the first review, the `Missing whitespace. 👾` message was because you need to add a space between the last `)` and the `{`.

Reply:
thanks McGiverGim - fixed!
@mikeller - OK no problem to stick with off, on, average2, average3 etc.

Reply:
@ctzsnooze: Ok, then we should change the lookup table elements to be indicative of what they actually represent: `OFF`, `ON`, `AVERAGED_2`, `AVERAGED_3`, ...

Reply:
I will do this prior to squashing and rebasing

Reply:
Averaging types renamed `AVERAGED_2`, `AVERAGED_3` etc..